test/integration/client-navigation/test/index.test.js

```diff
@@ -1748,6 +1748,50 @@ describe('Client Navigation', () => {
     expect(value).toBe(false)
   })
 
+  it.each([true, false])(
```

Comment:
```suggestion
  it.each(['true', 'false'])(
```

Reply:
I think this will break the assertion on line 1759 (because `"false"` != `false`)

(similarly for other comment)

Reply:
Yeah, just want to make sure `-${bool}` doesn't break in the future. Maybe you can change that one to `JSON.stringify`

Reply:
Sure, I've wrapped it with JSON.stringify.